sites/all/modules/custom/dccr_blocks/dccr_blocks.module

```diff
@@ -163,6 +173,14 @@ function dccr_blocks_block_view($delta = '') {
       $block = dccr_blocks_important_dates_block();
       break;
 
+    case 'dccr_final_schedule':
+      $block = dccr_get_final_schedule(FALSE, 'BOF', 'Schedule');
```

Comment:
I think empty_slot_text is not getting translatable. For solving this; please remove t() where you are finally printing that text and add t here.